### PR TITLE
OHM-296 list visualizers view

### PR DESCRIPTION
**Pull request by rcrichton**

### CODE REVIEW

File: app/scripts/controllers/visualizer.js [hunk 1]
```diff
@@ -4,25 +4,15 @@
 angular.module('openhimConsoleApp')
   .controller('VisualizerCtrl', function ($scope, $http, $interval, $window, login, Api, Alerting, Fullscreen) {
 
+    // initialize global variables
+    var settingsStore = {}; // a place the push current settings when switching to fullscreen
+    var visualizerUpdateInterval, updatePeriod, diffTime, lastUpdate, maxSpeed, pad;
+
     $scope.loadingVisualizer = true;
     $scope.loadingVisualizerError = false;
     $scope.loadingVisualizerErrorMsgs = [];
```
Comment (BMartinos) [inline app/scripts/controllers/visualizer.js]: The above two don't seem to be used anymore, but maybe they are kept in for future use?

File: app/views/visualizer.html [hunk 2]
```diff
@@ -10,48 +9,72 @@
           <div class="panel-title">
             <h2><i class="glyphicon glyphicon-road"></i>&nbsp;&nbsp;Visualizer</h2>
             <p class="text-muted">
-              The visualizer displays a live view of how transactions are being routed through the OpenHIM.
+              The visualizer displays a live view of how transactions are being routed through the OpenHIM. Multiple visualizers can be created and these are shared amoung admin users. Pick a visualizer on the left to start viewing or create a new one.
```
Comment (BMartinos) [inline app/views/visualizer.html]: Small typo: amoung

**Comment by BMartinos** [comment]
Hey @rcrichton The code looks and works great for me! Just some minor comments above. 

I am able to switch between Visualiser views easily and didn't encounter any issues. There are some styling changes we can make but not sure if its important to focus on those within this PR. 

The only other issue ive encountered, which might have been there for a while already is Firefox not rendering the Visualiser correctly. Screenshot below:
screenshot from 2016-08-11 09 50 43

File: app/scripts/controllers/visualizer.js (hunk 1 above)
Comment (rcrichton) [inline app/scripts/controllers/visualizer.js]: I believe they are still used, in line 52, 54, 66, 68 etc.

**Comment by rcrichton** [comment]
@BMartinos hmm interesting that firefox doesn't like the visualizer. I haven't touched the rendering code so I think this is an existing issue. Could you add a jira issue for this?

**Comment by BMartinos** [comment]
Just some of the styling comments i have:

1. List items border-radius should be removed to keep consistency with main styles
2. Reduce padding slightly to save some space
3. Move the "+ Visualiser" button to the top of the list so its easy to add a new ones
4. When the screen moves smaller ( responsive - 1300px ) the list moves above the Visualiser. This might need to be adapted into a show/hide or popup feature. thinking in terms of having 20+ visualisers and not being able to see it at the bottom of the list.

These styling comments could be tackled in a separate issue if they take to long as it doesnt affect the functionality of the View/Controller

File: app/scripts/controllers/visualizer.js (hunk 1 above)
Comment (BMartinos) [inline app/scripts/controllers/visualizer.js]: Ahhh yes you right, My search didn't pick them up sorry.

**Comment by rcrichton** [comment]
@BMartinos ok cool, I've adjusted the padding and the border-radius, thanks. I tried the '+ Visualizers' button at the top and it just looks weird. Maybe lets leave it as is for now and if it becomes a problem we can figure something out. Also, for the responsive stuff, I don't think there is a big use case for viewing the visualizer on mobile. I think the responsiveness is fine (nothing get displayed weird) and I don't want to add a bunch off new styling for something that won't really be used on mobile. Do you agree?

**Comment by BMartinos** [comment]
Great thanks @rcrichton !

I agree that we can leave out those responsive changes for now as its not important and wont really be used on mobile, We can also leave the "+ Visualisers" button where it is until we notice it becoming a problem. 

Just a very super tiny style for the nav-pill when hovering
.nav-pills > li > a:hover {
    border-radius: 0;
}

Everything works well so this is good to merge :+1:

**Comment by rcrichton** [comment]
@BMartinos cool, added that.

**Pull request merged**